### ISSUE TITLE
[Bug Report] list__tile__action--stack class applied with conditional rendering

### ISSUE THREAD

**Issue by bierik** [vuetifyjs/vuetify#4114]
### Versions and Environment
**Vuetify:** 1.1.0-alpha.6
**Vue:** 2.5.16
**Browsers:** Chrome 67.0.3396.40
**OS:** Mac OS 10.13.4

### Steps to reproduce
- Create simple v-list component with two v-btn inside v-list-tile-action
- Use v-if="true" and one button and v-if="false" on the other button

### Expected Behavior
The class list__tile__action--stack should not be applied to the single rendered button so it's displayed in the middle

### Actual Behavior
The list__tile__action--stack class is applied so the button renders on top of the list tile

### Reproduction Link
[https://codepen.io/bierik/pen/qYwdao](https://codepen.io/bierik/pen/qYwdao)

### Other comments
When using v-if="true" and v-else, it surprisingly working
There seems to be something wrong with the child count on https://github.com/vuetifyjs/vuetify/blob/v1.0.17/src/components/VList/VListTileAction.js#L8

**Comment by KaelWD** [comment]
Vue includes whitespace in `$children`, and also renders an empty comment for `v-if`
![image](https://user-images.githubusercontent.com/16421948/40459625-c8dd91b6-5f45-11e8-9804-3635b146bb66.png)
We should probably be filtering the array to not count those nodes.

**Comment by bierik** [comment]
So why is the child count equal to 1 when using `v-if="true"` and `else`.
```javascript
new Vue({
  template: `
<v-list two-line>
  <v-list-tile>
    <v-list-tile-content>
      <v-list-tile-title>some title</v-list-tile-title>
    </v-list-tile-content>
    <v-list-tile-action>
      <v-btn v-if="true">a</v-btn>
      <v-btn v-else>b</v-btn>
    </v-list-tile-action>
  </v-list-tile>
</v-list>
`
}).$mount('#app')
```

**Comment by KaelWD** [comment]
Because vue knows that you'll never have them both visible at the same time, so it just reuses the same element for both.

**Comment by ebisbe** [comment]
@KaelWD in the test the whitespace is not created. Not sure why.

**Comment by johnleider** [comment]
closed in #4657
